src/main/java/frc/utils/controllers/keyboard/KeyboardController.java

```diff
@@ -0,0 +1,152 @@
+        this.Z = new Trigger(new LoggedDashboardBoolean("Keyboard/z", false)::get);
+        this.X = new Trigger(new LoggedDashboardBoolean("Keyboard/x", false)::get);
+        this.C = new Trigger(new LoggedDashboardBoolean("Keyboard/c", false)::get);
+        this.V = new Trigger(new LoggedDashboardBoolean("Keyboard/v", false)::get);
+        this.B = new Trigger(new LoggedDashboardBoolean("Keyboard/b", false)::get);
+        this.N = new Trigger(new LoggedDashboardBoolean("Keyboard/n", false)::get);
+        this.M = new Trigger(new LoggedDashboardBoolean("Keyboard/m", false)::get);
+        this.COMMA = new Trigger(new LoggedDashboardBoolean("Keyboard/,", false)::get);
+        this.PERIOD = new Trigger(new LoggedDashboardBoolean("Keyboard/.", false)::get);
+        this.RIGHT_SHIFT = new Trigger(new LoggedDashboardBoolean("Keyboard/right shift", false)::get);
+        this.LEFT_CONTROL = new Trigger(new LoggedDashboardBoolean("Keyboard/ctrl", false)::get);
+        this.LEFT_ALT = new Trigger(new LoggedDashboardBoolean("Keyboard/alt", false)::get);
+        this.RIGHT_CONTROL = new Trigger(new LoggedDashboardBoolean("Keyboard/right ctrl", false)::get);
+        this.LEFT_ARROW = new Trigger(new LoggedDashboardBoolean("Keyboard/left", false)::get);
+        this.RIGHT_ARROW = new Trigger(new LoggedDashboardBoolean("Keyboard/right", false)::get);
+        this.UP_ARROW = new Trigger(new LoggedDashboardBoolean("Keyboard/up", false)::get);
+        this.DOWN_ARROW = new Trigger(new LoggedDashboardBoolean("Keyboard/down", false)::get);
+        this.NUMPAD_0 = new Trigger(new LoggedDashboardBoolean("Keyboard/numpad0", false)::get);
+        this.NUMPAD_1 = new Trigger(new LoggedDashboardBoolean("Keyboard/numpad1", false)::get);
+        this.NUMPAD_2 = new Trigger(new LoggedDashboardBoolean("Keyboard/numpad2", false)::get);
+        this.NUMPAD_3 = new Trigger(new LoggedDashboardBoolean("Keyboard/numpad3", false)::get);
+        this.NUMPAD_4 = new Trigger(new LoggedDashboardBoolean("Keyboard/numpad4", false)::get);
+        this.NUMPAD_5 = new Trigger(new LoggedDashboardBoolean("Keyboard/numpad5", false)::get);
+        this.NUMPAD_6 = new Trigger(new LoggedDashboardBoolean("Keyboard/numpad6", false)::get);
+        this.NUMPAD_7 = new Trigger(new LoggedDashboardBoolean("Keyboard/numpad7", false)::get);
+        this.NUMPAD_8 = new Trigger(new LoggedDashboardBoolean("Keyboard/numpad8", false)::get);
+        this.NUMPAD_9 = new Trigger(new LoggedDashboardBoolean("Keyboard/numpad9", false)::get);
+    }
+
+    public void runKeyboardToNetworkTables() {
+        Runtime rt = Runtime.getRuntime();
+        try {
+            System.out.println("py " + getPathToPython() + "keyboard_to_nt.py");
+            rt.exec(new String[]{"cmd.exe","/c","py " + getPathToPython() + "keyboard_to_nt.py"});
+
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public String getPathToPython() {
```

Comment:
![path to python](https://github.com/GreenBlitz/Off-Season-2024/assets/140002259/7a87c38b-696a-4f4a-893b-e845ddf9aac7)